.babelrc

```diff
@@ -25,6 +25,13 @@
         "transform-decorators-legacy"
       ]
     },
+    "test": {
+      "presets": [ "es2015-node4", "flow"],
```

Comment:
@juanpicado Actually I was trying to setup Jest from this branch. I am getting following error:
```
 SyntaxError: Unexpected token import
```
I think it is because of incorrect preset. When I change `es2015-node4` to es2015`, All the token error disapper and all tests also pass.

Please check.

Reply:
Well ... couple of things to mention here. 

Node4 does no support es2015 100%, so that preset is a Pollyfill. Now I was wondering whether make sense support Node 4 when the LTS dies next April. What do you think ?

Reply:
Also, we use a transpiler in this branch and the way we start in development verdaccio is different, please check dev section at wiki

Reply:
I think we can give support on Node 6. There is a very much difference from Node 4 to Node 6.

Reply:
Whats the different with "node4" prefix? the ES2015/env preset should "translate" all code into ES5, include "import" syntax.
By the way, some built-in function suchs as metods in Array.prototype need additional setup, babel-polyfill

Reply:
Well, `imports` for modules are not supported until `8.x` anyway, so, we need transpilers anyway. @Meeeeow we use `env` only for the UI you can check `.babelrc` and see we have many environments.  

I'm moving to Jest because of many reasons. There are some tricks at the functional testing folder in order to extend Promises and so on that trying to get rid of them, and those needs `es2015-node4`. 

Furthermore, these days I've been looking into the functional testing folder which is not migrated yet and the reasons are many. We need to run 3 servers (which in `master` branch are slow and fails constantly) which https://github.com/verdaccio/verdaccio/blob/feature-jest-test/test/helper/verdaccio-test depends on `require("babel-register");` which is even slower than the original implementation at `master`. I'm looking a better approach instead `fork` multiple processes which are not reliable. Tomorrow someone will come up with a 4th server and test will fail again. We need something more clean and maintainable. 

If you have something in mind, feel free to share your thoughts.